src/ejabberd_c2s.erl

```diff
@@ -2185,8 +2050,8 @@ check_privacy_and_route_or_ignore(Acc, StateData, From, To, Packet, Dir) ->
         _ -> Acc2
     end.
 
--spec resend_subscription_requests(mongoose_acc:t(), state()) -> {mongoose_acc:t(), state()}.
-resend_subscription_requests(Acc, #state{pending_invitations = Pending} = StateData) ->
+-spec resend_subscription_requests(mongoose_acc:t(), list(), state()) -> {mongoose_acc:t(), state()}.
```

Comment:
Does the `list()` has a known structure or is a list of anything?

Reply:
I'm getting more and more fond of static typing...